src/services/preference/preference.service.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { PreferenceService } from './preference.service';
+import { Observable } from 'rxjs';
+
+describe('PreferenceService', () => {
+  let preferenceService: PreferenceService;
+
+  beforeEach(() => {
+    preferenceService = new PreferenceService();
+  });
+
+  it('should instantiate.', () => {
+    expect(preferenceService).toBeInstanceOf(PreferenceService);
+  });
+
+  it('should have combined observable preferences.', done => {
+    expect(preferenceService.audification$).toBeInstanceOf(Observable);
+    preferenceService.audification$.subscribe(audificationPreference => {
+      Object.values(audificationPreference).forEach(value => {
+        expect(value).not.toBeInstanceOf(Observable);
```

Comment:
Might be helpful for future readers to quickly understand this method if there is also a simple assertion that audificationPreference equals an object of:
```
{
  enabled: true,
  lowestPitch: 256,
  ...
}
```
etc.

Reply:
I changed the structure of `PreferenceService` quite a bit in #35, so I'll address this when I update the tests.